src/cmd/sign.js

```diff
@@ -120,6 +120,8 @@ export default function sign(
         log.info('SUCCESS');
       } else {
         log.info('FAIL');
+        throw new UsageError(
```

Comment:
I thought more about this and I think it should be `WebExtError` because it may not always be something that the developer can fix. For example, signing might fail because there was a 500 response from the API, we really can't be sure.